src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -127,6 +130,18 @@ class ReplAutocompletion(globalContext: () => InterpreterContext) extends Comple
     }
   }
 
+  private object IsCommand {
```

Comment:
Because I needed the type alias `Command` which is defined in object `Commands`.